auth_proxy/models/user.py

```diff
@@ -16,3 +17,26 @@ class User(Base):
 
     id = Column(Integer, primary_key=True)
     patient_id = Column(String)
+    username = Column(String)
+    password = Column(String)
+    authenticated = Column(Boolean, default=False)
+
+    def is_active(self):
+        """ True, as all users are active.
+        """
+        return True
+
+    def get_id(self):
+        """ Return the user id.
+        """
+        return str(self.id)
+
+    def is_authenticated(self):
```

Comment:
I can't figure out where this property is set or the accessor accessed. Is there framework magic here?